src/site/content/en/blog/coop-coep/index.md

```diff
@@ -163,6 +177,13 @@ header to all documents including those that are embedded via iframes.
 parameters](https://first-party-test.glitch.me/coep).
 {% endAside %}
 
+{% Aside %}
+[squoosh.app](https://squoosh.app) [now uses COOP /
```

Comment:
```suggestion
[Squoosh](https://squoosh.app) (an image optimization PWA) [now uses COOP /
```
It's a bit awkward to have two links right next to each other so I'm just adding some text to add space between the links